platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/admin/ImportMigrationConfigurationAdminEntry.java

```diff
@@ -127,6 +127,14 @@ public String getPid() {
     return pid;
   }
 
+  public Dictionary<String, Object> getProperties() {
+    return properties;
+  }
+
+  public void setProperties(Dictionary<String, Object> properties) {
+    this.properties = properties;
+  }
```

Comment:
✏️ Could we expose a `getProperty(String propName)` and a `setProperty(String propName, Object propValue)` instead of exposing the whole dictionary?